tools/topology/topology2/cavs-src-mixin-mixout-hda.conf

```diff
@@ -33,6 +33,7 @@ Object.Pipeline {
 			index 2
 
 			Object.Widget.copier.1 {
+				node_type $HDA_LINK_OUTPUT_CLASS
```

Comment:
same comment, if node_type is a required attribute let's declare it as such in all the base pipeline classes.

Reply:
has to be part of copier class not pipeline class.